tests/unit/test_lookups.py

```diff
@@ -0,0 +1,230 @@
+import pytest
```

Comment:
unit and integration tests should replicate code structure these shouldn't be in a random file and should match where the lookups class is. i.e. it should be in 

`tests/unit/pipeline.py`